File: app/src/test/java/edu/gatech/donatrix/testLoginActivity.java
package edu.gatech.donatrix;

import org.junit.Test;

import java.util.HashMap;
import java.util.Map;

import edu.gatech.donatrix.model.User;

public class testLoginActivity {

    @Test
    public void testUserLogin() {
        Map<String, Object> body = new HashMap<>();
        body.put("email", "zday123@yahoo.com");
        body.put("password", "janisday");
//        assertEquals("User sign in correct", User.testLoggedIn(body), true);
    }
}

File: app/src/test/java/edu/gatech/donatrix/testLoginActivity.kt
package edu.gatech.donatrix

import org.junit.Test

import java.util.HashMap

import edu.gatech.donatrix.model.User

class testLoginActivity {

    @Test
    fun testUserLogin() {
        val body = HashMap<String, Any>()
        body["email"] = "zday123@yahoo.com"
        body["password"] = "janisday"
        //        assertEquals("User sign in correct", User.testLoggedIn(body), true);
    }
}